bitcoin-magazine/bitcoin-explained/bitcoin-core-0.21.0-new-release.md

```diff
@@ -0,0 +1,915 @@
+
+Aaron van Wirdum: 00:13:04
+
+So this was included in Bitcoin Core 21. What does that mean exactly, to have it included in Bitcoin Core 21?
+
+Sjors Provost: 00:13:11
+
+Well, it means that you can start Bitcoin Core with the Dash SIGNET, just like you can start it with dash testnet and your icon will have a nice different color and there's a faucet out there that you can go to and you can get signet coins which you get by the dozen I think so it's nice yeah you can use it yeah and we'll go back to it again in the well I guess we can skip to that item now because it makes sense. Because the other thing that's included is Taproot.
+
+Aaron van Wirdum: 00:13:36
+
+Right yeah okay.
+
+Sjors Provost: 00:13:38
+
+And Taproot is active on Signet.
+
+Aaron van Wirdum: 00:13:41
+
+Yeah so we now have Taproot on Signet.
+
+Sjors Provost: 00:13:43
+
+That's right.
+
+Aaron van Wirdum: 00:13:44
+
+Obviously this is meaningless for regular users, this is just for developers, just for testing their software.
+
+Sjors Provost: 00:13:51
+
+Exactly, but the main plan was to have the taproot code inside of Bitcoin Core, unactivated, but just there so people can find bugs in it and can play with it.
+
+Aaron van Wirdum: 00:13:59
+
+Which is also the case, it's in there now.
+
+Sjors Provost: 00:14:02
+
+Exactly, so it's nice because normally you would have the code of a soft fork in there and you could theoretically play with it but it's not very clear how you would do that but now it's very clear because you can use it on Signet and the nice thing about Signet is that the rules can be changed anytime because there's you know two miners that can centralize decide to just enforce different taproot rules if the taproot spec is changed because it's not final until it's live
+
+Aaron van Wirdum: 00:14:29
+
+yeah well it's kind of final right The idea is that from here on out, it shouldn't be changed anymore, I think.That's why it's included in the code in the first place.
+
+Sjors Provost: 00:14:38
+
+Yeah, I think you'd see it as a release candidate.
+
+Aaron van Wirdum: 00:14:40
+
+Right.
+
+Sjors Provost: 00:14:41
+
+Because even if Taproot activates on mainnet in the future with a new version of Bitcoin Core, this old version of Bitcoin Core won't do anything with that.
+
+Aaron van Wirdum: 00:14:49
+
+Right, yeah.
+
+Sjors Provost: 00:14:50
+
+So, even if the rules change now, there's no problem with your 0.21 node.It will not get confused if Taproot changes. Of course, hopefully it doesn't.
+
+Aaron van Wirdum: 00:14:59
+
+Sure, But about that, so yeah, the taproot code is included in Bitcoin Core 21.
+This also means that probably the next minor version of Bitcoin Core, which would be Bitcoin Core 21.1, well, Bitcoin Core 0.21.1, will have the actual activation logic for TapRoot.
```

Comment:
"[taproot](https://bitcoinops.org/en/topics/taproot/)" is the correct way of writing it, not "TapRoot".